pkg/skaffold/test/test_factory_test.go

```diff
@@ -121,6 +123,11 @@ func TestTestSuccess(t *testing.T) {
 			CmdRun("container-structure-test test -v warn --image image:tag --config "+tmpDir.Path("tests/test1.yaml")+" --config "+tmpDir.Path("tests/test2.yaml")).
 			AndRun("container-structure-test test -v warn --image image:tag --config "+tmpDir.Path("test3.yaml")))
 
+		t.Override(&structure.GetImagefn, func(context.Context, io.Writer, string, []build.Artifact, docker.LocalDaemon,
```

Comment:
consider moving these test to pkg/skaffold/structure_test.go if its testing container structure test flow.

Reply:
This test is testing e2e workflow of test cases. It is not specific to structure tests.